### PR TITLE
Lucas/t 295 fix checkbox column width

### LINKED ISSUE #262: Fix checkbox column width
Reduce checkbox column width & remove right border 

Online: 

image

Figma

image

### PR DESCRIPTION
**Pull request by lucasbordeau**
- Fix checkbox column width
- Removed checkbox column borders
- Fixed table width autoresize (couldn't fix size of columns)
- Defined minimum and maximum width for columns

### CODE REVIEW

**Comment by lucasbordeau** [comment]
This PR breaks editable cells

File: front/src/modules/ui/components/table/EntityTable.tsx [hunk 1]
```diff
@@ -53,26 +51,55 @@ const StyledTable = styled.table`
     padding: 0;
     border: 1px solid ${(props) => props.theme.tertiaryBackground};
     text-align: left;
+
+    min-width: ${(props) => props.theme.table.cellMinWidth};
```
Comment (charlesBochet) [inline front/src/modules/ui/components/table/EntityTable.tsx:55]: remove as its conflicts with long term vision

File: front/src/modules/ui/components/table/EntityTable.tsx [hunk 2]
```diff
@@ -154,22 +182,30 @@ export function EntityTable<
...
                 ))}
+                <th></th>
               </tr>
             ))}
           </thead>
           <tbody>
             {table.getRowModel().rows.map((row, index) => (
               <tr key={row.id} data-testid={`row-id-${row.index}`}>
-                {row.getVisibleCells().map((cell) => {
+                {row.getVisibleCells().map((cell, cellIndex) => {
```
Comment (charlesBochet) [inline front/src/modules/ui/components/table/EntityTable.tsx:192]: remove cellIndex

File: front/src/modules/ui/components/table/EntityTable.tsx [hunk 3]
```diff
@@ -139,11 +166,12 @@ export function EntityTable<
           <thead>
             {table.getHeaderGroups().map((headerGroup) => (
               <tr key={headerGroup.id}>
-                {headerGroup.headers.map((header) => (
+                {headerGroup.headers.map((header, headerIndex) => (
```
Comment (charlesBochet) [inline front/src/modules/ui/components/table/EntityTable.tsx:169]: remove headerIndex

File: front/src/modules/ui/components/table/EntityTable.tsx [hunk 4]
```diff
@@ -53,26 +51,55 @@ const StyledTable = styled.table`
...
+    min-width: ${(props) => props.theme.table.cellMinWidth};
+    max-width: ${(props) => props.theme.table.cellMaxWidth};
+
     text-align: left;
+
     :last-child {
       border-right-color: transparent;
     }
     :first-of-type {
+      min-width: ${(props) => props.theme.table.checkboxColumnWidth};
```
Comment (charlesBochet) [inline front/src/modules/ui/components/table/EntityTable.tsx:91]: remove

**Review by charlesBochet — APPROVED** [review]

**Pull request merged**